### ISSUE TITLE
Unnecessary changes on update

### ISSUE THREAD

**Issue by tulimaki** [suutari/prequ#30]
It seems that some package names are randomly either with or without capital letter.

For me this change is introduced pretty much every prequ update.
```
-markupsafe==1.0           # via jinja2
+markupsafe==1.0           # via Jinja2
```

**Comment by suutari** [comment]
This should be fixed in Prequ 1.4.3 which was released 2 days ago.

**Comment by tulimaki** [comment]
Got it. Thanks!
